### PR TITLE
Remove APIs marked for deprecation after 2023 10 01

### PR DESCRIPTION
**Pull request by vEpiphyte**
- Remove all APIs documented for deprecation after 20231001 https://synapse.docs.vertex.link/en/latest/synapse/changelog.html#api-deprecation-notice-2023-10-01
- Remove all functionality related to the deprecated APIs. This includes the following:
  - HTTP API `/api/v1/storm` now produces `node:edits` messagse instead of `splices` by default.
  - HTTP API `/api/v1/watch` is removed. This API was not documented.
  - Cmdr `storm` command no longer displays splices.
  - Cmdr `log` command no longer records splices.
   - `synapse.tools.csvtool` no longer records or displays splices
   - `synapse.tools.feed` no longer supports splices or nodeedits as input and the splice documentation example has been removed.
- Remove `s_common.aclosing()` as we are past is EOL version.

### CODE REVIEW

File: docs/synapse/userguides/syn_tools_feed.rst [hunk 1]
```diff
@@ -118,114 +122,22 @@ From which we can issue Storm commands to interact with and validate the nodes t
...
-
-**Loading the Data:**
-
-As before, once the data has been inspected and approved, we can point the ``feed`` tool at the Cortex we want to apply the splices to in order to apply them.
-
-::
-
-    python -m synapse.tools.feed --cortex tcp://cortex.vertex.link:4444/cortex00 --format 'syn.splice' 
-      testsplices.yaml
+With the ``chunksize`` parameter signifying that the ``feed`` tool should read two lines at a time from the file and
```
Comment (MichaelSquires) [inline docs/synapse/userguides/syn_tools_feed.rst:140]: `--chunksize 1` means read two lines at a time? Is that correct? Seems like `--chunksize 2` should mean two lines at a time!?

File: synapse/tests/test_lib_storm.py [hunk 2]
```diff
@@ -4816,3 +4512,34 @@ async def test_lib_dmon_embed(self):
             ''')
 
             self.none(await core.callStorm('return($lib.queue.gen(haha).get().1)'))
+
+    # async def test_storm_onload_persistence(self):
```
Comment (MichaelSquires) [inline synapse/tests/test_lib_storm.py:4516]: Is this commented on purpose or an artifact that you forgot to uncomment/delete?

**Review by MichaelSquires — CHANGES_REQUESTED** [review]
Wow, that was a lot of cutting! 

Looks pretty good, just a couple small comments/questions.

File: docs/synapse/userguides/syn_tools_feed.rst (hunk 1 above)
Comment (vEpiphyte) [reply docs/synapse/userguides/syn_tools_feed.rst:140]: There is some non-intuitive behavior here. Will clean those up separately.

**Review by MichaelSquires — DISMISSED** [review]

File: docs/synapse/userguides/syn_tools_feed.rst [hunk 3]
```diff
@@ -29,13 +29,12 @@ Where:
   
 - ``FORMAT`` specifies the format of the input files. 
 
-  - Currently, only the values "syn.nodes", "syn.splices", and "syn.nodeedits" are supported.
-  - Defaults to "syn.nodes" if not specified
-  
+  - Currently, only the value "syn.nodes". This is also the default value.
```
Comment (Cisphyx) [inline docs/synapse/userguides/syn_tools_feed.rst:32]:
```suggestion
  - Currently, only the value "syn.nodes" is supported. This is also the default value.
```

**Review by Cisphyx — APPROVED** [review]

**Review by invisig0th — APPROVED** [review]

**Pull request merged**